### DISCUSSION TITLE
What's a good way to attach an external antenna to the repeater?

### DISCUSSION THREAD

**Discussion by mortenmoulder** [basilfx/TRADFRI-Hacking#65] [Q&A]
I disassembled one of the repeaters and found an ICC-A-1 inside. I was hoping I could increase the range, by attaching an external antenna, but I'm not sure how one would do that (I have little to no experience with antennas).

I do have a couple of different antennas laying around, though. From good routers and some ESP32 ones, as well as some for LTE routers.

I am willing to sacrifice basically any of these antennas, by trying to attach it to one of my IKEA repeaters. However, to my knowledge, simply adding an antenna won't do anything, because the transmit power is still the same.

I want to get the maximum range possible from one of these. How would I go about doing that?

**Comment by MattWestb** [comment]
The ICC-A-1 module RF part is tuned for the PCB antenna so its no 110% match for one external one.
You can tying cutting the PCB line after the golden part that is exposed and soldering the antenna there and finding so good grounding for the grout path but i think you is getting the resolute that many with CC-2531 sticks have that the  internal antenna is working better then the external because its not being matched to the external 50 Ohm antenna.

ESP32 antennas is working on the same frequency = OK is good quality.
Other LTE, 434 and so on it not matched and is very likely working very bad.

If you is having one device with the new Silabs module you can changing the antenna path and using the pads for the external antenna and all is OK and very power full !!
